behavioral-questions.md

```diff
@@ -4,16 +4,16 @@ Answer the following questions. Use the Nugget + STAR method for each answer.
 
 1. Why did you choose a career in software development?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
+- Nugget: I chose a career in software development because of my passion for learning, keen eye for detail and  commitment to enhancing efficiency.
+- Situation: In my academic and life experiences, I have found myself naturally drawn to things that improve efficiency. I started a family business that requires a platform and web presence. None of the software developers that I contacted were able to help me to bring my vision for the family business to pass, so I decided to learn how to code so that I can create the platform for myself. 
+- Task: Recognizing everyone’s inability to bring my vision to pass, I decided to explore a career in software development. I researched available options in my community as well as options outside of the Arizona area. The task at hand was to gain admission to a reputable school and complete the course and secure employment within a reasonable amount of time, specifically before my son is born.
```

Comment:
I really like how you have focused on this answer and dug into what inspired you.  The one thing I would consider rewording if this was asked in an interview is the part where you mention recognizing everyone's inability to bring your vision to pass.  This could be looked at negatively in an interview.  Hiring managers will often look at how you perceive and talk about others as well as yourself to see how or if you will fit in with their culture.